dbt/dbt_project.yml

```diff
@@ -59,5 +59,7 @@ models:
       +schema: sale
     qc:
       +schema: qc
+    ephemeral:
+      +schema: ephemeral
```

Comment:
**[Suggestion, non-blocking]** If we're going to define an `ephemeral` schema anyway, it might make sense to use this opportunity to set `materialized: ephemeral` for all models in this schema, which would eliminate the need for setting `config(materialized = 'ephemeral')` in every ephemeral query file. But I could also see an argument that the explicitness of the `config(materialized = 'ephemeral')` call might be helpful for readers, given that this type of materialization is relatively uncommon compared to `view` and `table`.

Reply:
Agreed! I think it's totally sensible to just make everything under that schema ephemeral by default. Changed.